doc/how-to/ovn_underlay.md

```diff
@@ -0,0 +1,83 @@
+(howto-ovn-underlay)=
+# How to configure an OVN underlay network
+
+When running {command}`microcloud init`, if you decided to setup distributed networking and if you have enough at least one unused network interface per cluster member with an IP address, you are asked if you want to configure an underlay network for OVN.
+Here is the question you will be asked:
+
+`Configure dedicated underlay networking? (yes/no) [default=no]: <answer>`
+
+You can choose to skip this question (just hit `Enter`). This will then use the internal MicroCloud network as an OVN 'underlay', which is the same as the OVN management network ('overlay' network).
+
+You could also choose to configure a dedicated underlay network for OVN with typing `yes`. A list of available network interfaces with an IP address will be displayed.
+You can then select one network interface per cluster member to be used as the interfaces for the underlay network of OVN.
+
+The following instructions build on the {ref}`get-started` tutorial and show how you can test setting up a MicroCloud with an OVN underlay network.
+
+1. Create the dedicated network for the OVN underlay:
+
+   1. First, create a dedicated network for the OVN cluster members to be used as an underlay. Let's call it `ovnbr0`:
+
+           lxc network create ovnbr0
+
+   1. Enter the following commands to find out the assigned IPv4 and IPv6 addresses for the networks and note them down:
+
+           lxc network get ovnbr0 ipv4.address
+           lxc network get ovnbr0 ipv6.address
+
+1. Create the network interfaces that will be used for the OVN underlay setup for each VM:
+
+   1. Add the network device for the `ovnbr0` network:
```

Comment:
No need for a substep here - just combine the two intros into one.

Reply:
fixed